Add docs to mention changes regarding modular upgrades (#5932)
aws/eks-anywhere: docs/content/en/docs/reference/support/support-versions.md

### Will I get notified when there is a new Amazon EKS Anywhere version release?
You will get notified if you have subscribed as documented on the [Release Alerts]({{< relref "../snsupdates/" >}}) page.

### Can I use different Amazon EKS Anywhere minor versions for my management cluster and workload clusters?

> Yes, the management cluster and its workload clusters can be created using different Amazon EKS Anywhere minor versions.

Reviewer: ```suggestion
Yes, the management cluster can be upgraded to newer Amazon EKS Anywhere versions than its workload clusters.
```

Technically you can't create a workload cluster on a version if the management cluster was never in that version. Example: create management cluster with v0.16.0. You can create workload clusters with v0.16.0 but not with v0.15.0. On the contrary, if you create your management cluster with version v0.15.0 and then upgrade to v0.16.0, you can create workload clusters in either v0.16.0 or v0.15.0.

Not sure if we want to add all this detail

Author: I definitely can see this coming up as a question from users, so I added a note with those details

However, we provide a maximum skew version support of one EKS Anywhere minor version for management and workload clusters.
This means that we support the management cluster being one EKS Anywhere minor version newer than the workload clusters (such as v0.16 for workload clusters if the management cluster is at v0.17).

> In the event that you want to upgrade your management cluster to a version that does not satisfy this condition, we recommend upgrading the workload cluster's EKS-A version first, followed by upgrading to your desired EKS-A version for the management cluster.

Reviewer 2: ```suggestion
In the event that you want to upgrade your management cluster to a version that does not satisfy this condition, we recommend upgrading the workload cluster's EKS Anywhere version first, followed by upgrading to your desired EKS Anywhere version for the management cluster.
```
